Tests/CommandsTests/WorkspaceTests.swift

```diff
@@ -386,67 +386,74 @@ final class WorkspaceTests: XCTestCase {
                     MockPackage("A", version: nil), // To load the edited package manifest.
                 ]
             )
-            // Create the workspace.
-            let workspace = try Workspace(rootPackage: path, manifestLoader: manifestGraph.manifestLoader, delegate: TestWorkspaceDelegate())
```

Comment:
Technically, this isn't really NIFC since it really does change the functionality (I assume it broke otherwise).